src/langcheck/augment/en/_keyboard_typo.py

```diff
@@ -4,13 +4,27 @@
 
 
 def keyboard_typo(
-    texts: list[str] | str,
+    instances: list[str] | str,
+    num_perturbations: int = 1,
     **kwargs,
 ) -> list[str]:
-    '''Generate keyboard typo perturbed texts for augmentation.
+    '''Applies a keyboard typo text perturbation to each string in instances
+    (usually a list of prompts).
 
     Args:
-        texts: List of texts to be augmented.
+        instances: A single string or a list of strings to be augmented.
+        num_perturbations: The number of perturbed instances to generate for
+            each string in instances
+        aug_char_p: Percentage of characters (per token) that will be augmented.
```

Comment:
I moved the definition of default values to `Args:` instead of the note. This is consistent with `synonym()` and is easier to notice